### DISCUSSION TITLE
Full-width images are blurry after update.

### DISCUSSION THREAD

**Discussion by benzitney** [GitbookIO/community#566] [2. Published content]
After the update, something is going wrong when loading images in full width.

In the editor, I have loaded the following image to the page:
https://files.gitbook.com/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FWDvO31PP3F01d6jisBTt%2Fuploads%2F2YVQa1IbyOz4HLSkQnEm%2Fimage.png?alt=media&token=6eef3fad-1d72-4f94-b751-5be411a8f14e

When rendered in the published page in a full-width image block, however, it loads only a 768px-wide image:
https://help.daasity.com/~gitbook/image?url=https:%2F%2F1080510223-files.gitbook.io%2F%7E%2Ffiles%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252FWDvO31PP3F01d6jisBTt%252Fuploads%252F2YVQa1IbyOz4HLSkQnEm%252Fimage.png%3Falt=media%26token=6eef3fad-1d72-4f94-b751-5be411a8f14e&width=768&dpr=1&quality=100&sign=bf8255ae5e5d34fe8d6e8c105eb44002b919d6010b2c4c82f833928a097dc1e9

Since I'm on a larger monitor, it means gitbook is loading a 768px-wide image into a div that's over 1200px wide, so it naturally looks blurry.

For whatever reason, if I click on the blurry image to zoom, the image then gets replaced with an appropriately sized version of the image (2300px wide), which looks great: https://help.daasity.com/~gitbook/image?url=https:%2F%2F1080510223-files.gitbook.io%2F%7E%2Ffiles%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252FWDvO31PP3F01d6jisBTt%252Fuploads%252F2YVQa1IbyOz4HLSkQnEm%252Fimage.png%3Falt=media%26token=6eef3fad-1d72-4f94-b751-5be411a8f14e&width=768&dpr=4&quality=100&sign=bf8255ae5e5d34fe8d6e8c105eb44002b919d6010b2c4c82f833928a097dc1e9

But upon initial loading, it is still the small version and looks very bad. This is happening for all full-width images, not just this one.

The page for reference is https://help.daasity.com/extract/integrations/amazon-seller-central/validating-your-amazon-data-in-daasity and I'm viewing on a 24-inch screen

**Comment by LegendTheDeveloper** [comment]
Up
